puppet/zulip/files/logrotate/zulip

```diff
@@ -1,8 +1,13 @@
-/var/log/zulip/server.log /var/log/zulip/workers.log /var/log/zulip/manage.log {
+/var/log/zulip/*.log {
 	missingok
 	rotate 10
 	size 1G
 	compress
 	delaycompress
 	notifempty
+        create 644 zulip zulip
```

Comment:
I think we had issues, but the fix was put in there some time ago... so I don't remember sadly.

I don't think it would hurt leaving it there... so unless you see any side effect I'd actually leave it in there.